components/dashboard/src/projects/NewProject.tsx

```diff
@@ -214,9 +214,11 @@ export default function NewProject() {
     const renderSelectRepository = () => {
 
         const noReposAvailable = reposInAccounts.length === 0;
-        const filteredRepos = Array.from(reposInAccounts).filter(r => r.account === selectedAccount && r.name.includes(repoSearchFilter));
+        const filteredRepos = Array.from(reposInAccounts).filter(r => r.account === selectedAccount && `${r.name}`.toLowerCase().includes(repoSearchFilter.toLowerCase()));
```

Comment:
**praise**: Thanks so much for fixing this! I was wondering why some projects were not included in the search results when using some lowercase letters. ❗